archimedes/Dockerfile

```diff
@@ -2,9 +2,9 @@ FROM archimedes-base
 # Perform these steps first to allow better caching behavior
 #RUN bundle install
 WORKDIR /app
-COPY src/poetry.lock /app/poetry.lock
```

Comment:
The way `build_image` used to work was that it required Dockerfiles to refer to their own working directory as `src` to distinguish it from other dependent source directories.  This is no longer the case.  You'll see how I solved this by using multi-stage builds in other files later on.